dango/types/src/warp.rs

```diff
@@ -85,6 +86,18 @@ pub struct Route {
     pub fee: Uint128,
 }
 
+// -------------------------------- rate-limit ---------------------------------
+
+#[grug::derive(Serde)]
+pub struct RateLimitBound;
+
+impl Bounds<Udec128> for RateLimitBound {
+    const MAX: Option<Bound<Udec128>> = Some(Bound::Inclusive(Udec128::ONE));
+    const MIN: Option<Bound<Udec128>> = Some(Bound::Exclusive(Udec128::ZERO));
+}
+
+pub type RateLimit = Bounded<Udec128, RateLimitBound>;
```

Comment:
You can just use `grug::ZeroInclusiveOneExclusive` for this. Read my issue: https://github.com/left-curve/left-curve/issues/456